homebrew_challenge/src/App.jsx

```diff
@@ -377,56 +377,124 @@ const App = () => {
+      },
+      {
+        "name": "Craig Climie",
+        "linkedinRef": "https://www.linkedin.com/in/craig-climie"
+      }
+    ];
+
+    const sortedTeamMembers = teamMembers.sort((a, b) => a.name.localeCompare(b.name)).map((data, index) => {
+      return(
+        <li key={index}>
           <a
-            href={linkedInRef}
+            href={data.linkedinRef}
             target="_blank"
             rel="noopener noreferrer"
-            className="link "
+            className="link"
           >
-            {name}
+            {data.name}
           </a>
         </li>
-      );
-    }
+      )
+    });
 
     return (
-      <div fluid className="about-us">
-        <p>
-          The COVID-19 Dashboard for Scotland was created by the following STA
-          volunteers:
-        </p>
-        <ul>
-          {person(
-            "Rhi Batstone",
-            "https://www.linkedin.com/in/rhiannon-batstone-076191120"
-          )}
-          {person("Ric Clark", "https://www.linkedin.com/in/richard--clark")}
-          {person("Craig Climie", "https://www.linkedin.com/in/craig-climie")}
-          {person("Jonathan Lau", "https://www.linkedin.com/in/jonathancylau")}
-          {person("Colin Lyman", "https://www.linkedin.com/in/colin-lyman")}
-          {person(
-            "Cristina Perez",
-            "https://www.linkedin.com/in/cristina-perez-11229846"
-          )}
-          {person(
-            "Luke Pritchard-Woollett",
-            "https://www.linkedin.com/in/lukepritchardwoollett"
-          )}
-          {person(
-            "Euan Robertson",
-            "https://www.linkedin.com/in/euan-robertson-5845582"
-          )}
-          {person(
-            "Gabriela Satrovskaja",
-            "https://www.linkedin.com/in/gabriela-satrovskaja"
-          )}
-          {person("Allan Stevenson", "https://www.linkedin.com/in/alstev")}
-          {person("Donal Stewart", "https://www.linkedin.com/in/donalstewart")}
-          {person("Becky Still", "https://www.linkedin.com/in/rebeccastill1")}
-        </ul>
+      <div fluid="true" className="about-us">
+        <hr/>
+        <h1>About us</h1>
+        <p>This dashboard has been developed by members of the Scottish Tech Army to improve awareness of the impacts of Covid-19.</p>
+        <h2>The Scottish Tech Army</h2>
+        <p>Founded by Edinburgh based entrepreneurs, Alistair Forbes and Peter Jaco, the Scottish Tech Army Limited is a not for profit company that is building a volunteer Covid-19 technical response team that will work to help the Scottish Government, Local Authorities and other organisations across the country with rapid technical development projects to address current Covid-19 related challenges and post pandemic economic recovery. </p>
+        <h2>Meet the team</h2>
+        <p>The Covid-19 dashboard for Scotland was created by the following STA volunteers: </p>
+        {window.innerWidth > 780 ? (
```

Comment:
We have multiple ways of specifying responsive behaviour in this app. I think we should standardise on one - not necessarily in this PR though

Reply:
Also, is this even needed ? At its minimum width, there is no change from 2 column in my browser (Firefox). It still looks fine in 2 column mode.

Reply:
It's kinda tricky tho, if you open the website on a device with smaller screen, the changes will happen (one column only). But when you are moving around the "borders" in inspect mode nothing happens. But if you think we don't need to change the number of columns I'm happily deleting that piece of code!

Reply:
Removed it, I thought it wouldn't look great in separated columns on a smaller screen, but it's not bad!